ts/data.ts

```diff
@@ -7,21 +7,31 @@ interface DataInterface {
 }
 const data: DataInterface = {
   view: 'entry-form',
-  entries: [],
+  entries: readData(),
   editing: null,
   nextEntryId: 1,
 };
 
 function writeData(): void {
   const dataJSON = JSON.stringify(data.entries);
+  const dataViewJSON = JSON.stringify(data.view);
+  localStorage.setItem('data-view', dataViewJSON);
   localStorage.setItem('data-storage', dataJSON);
 }
 
-function readData(): string[] {
+function readData(): CodeJournalForm[] {
   const dataStorage = localStorage.getItem('data-storage');
   if (typeof dataStorage === typeof '' && dataStorage !== null) {
     return JSON.parse(dataStorage);
   } else {
     return [];
   }
 }
```

Comment:
This should be returning the object in local storage or the default state of the object if there is none